src/main/java/com/microsoft/sqlserver/jdbc/SQLServerPreparedStatement.java

```diff
@@ -63,6 +63,9 @@
     /** SQL statement with expanded parameter tokens */
     private String preparedSQL;
 
+    /** True if this execute has been called for this statement at least once */
+    private boolean bExecutedAtLeastOnce = false;
```

Comment:
I saw the other field above called nBatchStatementDelimiter and thought there was a hungarian thing going on in this file, looking around more seems like not. Renamed to isExecutedAtLeastOnce.